target/linux/realtek/image/rtl930x.mk

```diff
@@ -114,15 +114,14 @@ define Device/zyxel_xgs1210-12-a1
 endef
 TARGET_DEVICES += zyxel_xgs1210-12-a1
 
-define Device/zyxel_xgs1250-12-a1
+define Device/zyxel_xgs1250-12-common
```

Comment:
This breaks because you make the Makefile look for a zyxel_xgs1250-12-common DTS, which does not exist. Define the common code in `rtl9302_zyxel_xgs1250-12-common.dtsi` like is done for the GS1900-xx e.g. Then use `$(Device/zyxel_gs1900)` to call the recipe from the rtl930x Makefile.

I'm not sure if it's worth it to split this out for two devices, but I'll let others be the judge of that.

Reply:
Nevermind me. I'd still move it out to `target/linux/realtek/image/common.mk` though for the sake of consistency and cleanliness. That's where the other shared device recipes are all concentrated.

Reply:
Did not see that. Thought you changed back the wrong way. Leave as is to align with file. Thanks.